3drepo/3drepo.io
# ISSUE #3406 - Unexpected behaviour in upload revisions modal

This fixes #3406 

#### Description
- Cleaned up a lot of code
- Form is cleared when modal is closed
- The latest revision in the Destination dropdown now shows the latest revision instead of the container name
- Revision description errors are now visible
- Modal responds better to smaller screen sizes
- Long container names are now truncated in destination list
- Removed console errors when using autocomplete
- **FIXED**: Selecting a destination, clearing the selection, and then trying to choose the initial destination again doesn't work. It says it's already in use.
- **FIXED**: Selecting an existing container. Then changing to creating a new container doesn't disable the sidebar fields.

#### Test cases
- Open the uploads modal
- Try uploading files
- See how smaller screens look in both stages of the upload screen
